src/app/functions.js

```diff
@@ -0,0 +1,201 @@
+}
+
+const threeOfKindPoints = numbers => {
+  return isThreeOfKind(numbers) ? sumAll(numbers) : 0
+}
+
+export const isFourOfKind = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5 || isFullHouse(numbers)) return false
+  let sameNumbers = 0
+  sortedArray.reduce((a, b) => {
+    if (a === b) sameNumbers = sameNumbers + 1
+    return b
+  })
+  return sameNumbers === 3
+}
+
+const fourOfKindPoints = numbers => {
+  return isFourOfKind(numbers) ? sumAll(numbers) : 0
+}
+
+export const isFullHouse = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5) return false
+  const isFirstTwoAreSame = sortedArray[0] === sortedArray[1]
+  const isLastTwoAreSame = sortedArray[3] === sortedArray[4]
+  const isFirstThreeAreSame = isFirstTwoAreSame && sortedArray[1] === sortedArray[2]
+  const isLastThreeAreSame = isLastTwoAreSame && sortedArray[2] === sortedArray[3]
+  if (isFirstThreeAreSame && isLastTwoAreSame && sortedArray[2] !== sortedArray[3]) {
+    return true
+  } else if (isFirstTwoAreSame && isLastThreeAreSame && sortedArray[1] !== sortedArray[2]) {
+    return true
+  } else {
+    return false
+  }
+}
+
+const fullHousePoints = numbers => {
+  return isFullHouse(numbers) ? 25 : 0
+}
+
+export const isSmallStraight = numbers => {
+  // 1-2-3-4
+  // 2-3-4-5
+  // 3-4-5-6
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5) return false
+  let consecutiveNumbers = 0
+  sortedArray.reduce((a, b) => {
+    if (a + 1 === b) consecutiveNumbers = consecutiveNumbers + 1
+    return b
+  })
+
+  return consecutiveNumbers === 3
+}
+
+const smallStraightPoints = numbers => {
+  return isSmallStraight(numbers) ? 30 : 0
+}
+
+export const isLargeStraight = numbers => {
+  // 1-2-3-4-5
+  // 2-3-4-5-6
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5) return false
+  let consecutiveNumbers = 0
+  sortedArray.reduce((a, b) => {
+    if (a + 1 === b) consecutiveNumbers = consecutiveNumbers + 1
+    return b
+  })
+
+  return consecutiveNumbers === 4
+}
+
+const largeStraightPoints = numbers => {
+  return isLargeStraight(numbers) ? 40 : 0
+}
+
+export const isYahtzee = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5) {
+    return false
+  }
+  let sameNumbers = 0
+  sortedArray.reduce((a, b) => {
+    if (a === b) sameNumbers = sameNumbers + 1
+    return b
+  })
+  return sameNumbers + 1 >= 5
+}
+
+const yahtzeePoints = numbers => {
+  return isYahtzee(numbers) ? 50 : 0
+}
+
+export const isChance = numbers => {
+  const sortedArray = numbers.sort((a, b) => a > b)
+  if (sortedArray.length > 5) return false
+  if (isFullHouse(numbers)) return true
+  let sameNumbers = 0
+  sortedArray.reduce((a, b) => {
+    if (a === b) sameNumbers = sameNumbers + 1
+    return b
+  })
+  return sameNumbers >= 1
```

Comment:
You're checking against `sameNumbers >= 1`. This is always true, or?

I think this line of code does the same like your complete algorithm:
`return sortedArray.length <= 5;`